src/eth1spec/evm/interpreter.py

```diff
@@ -0,0 +1,86 @@
+# flake8: noqa 405
+
+"""
+EVM Interpreter
+------------------------------
+
+A straightforward interpreter that executes EVM code.
+"""
+
+from dataclasses import dataclass
+from typing import List, Tuple
+
+from ..eth_types import EMPTY_ACCOUNT, U256, Address, Hash32, Log, State, Uint
+from . import Environment, Evm
+from .gas import GAS_VERY_LOW
+from .ops import *  # noqa: F403
```

Comment:
This really limits the effectiveness of the linter, so we should avoid star imports.

Reply:
Okay :(